bosh-setup/azuredeploy.json

```diff
@@ -89,19 +66,42 @@
     },
     "enableDNSOnDevbox": {
       "type": "bool",
+      "defaultValue": true,
       "allowedValues": [
         true,
         false
       ],
       "metadata": {
-        "description": "A default DNS will be setup in the devbox if it is true. Need a manual reboot to take effect."
+        "description": "A default DNS will be setup in the devbox if it is true."
+      }
+    },
+    "tenantID": {
```

Comment:
Can you please include the link in the description tag, it's going to be hard to discover the guide for anyone who's already downloaded the template or deploying from Azure.com. Additionally, it might be worth it to include the guide inside the readme here as well to reduce the hoops a customer has to go through before deploying this template